### ISSUE TITLE
Leptos Book - 02_getting_started - 

### ISSUE THREAD

**Issue by jacksonbrim** [leptos-rs/leptos#1257]
**Describe the bug**
When following along with [the book](https://github.com/leptos-rs/leptos/blob/main/docs/book/src/02_getting_started.md)
The browser doesn't render the expected "Hello, world!" and instead returns a blank page.

**Leptos Dependencies**
Cargo.toml: 
```
[package]
name = "leptos-tutorial"
version = "0.1.0"
edition = "2021"

[dependencies]
leptos = "0.4.1"
```

**To Reproduce**
```shell
cargo install trunk
cargo init leptos-tutorial

rustup toolchain install nightly
rustup default nightly
rustup target add wasm32-unknown-unknown

cargo add leptos```

Follow 02_getting_started steps:
Create index.html in the root of leptos-tutorial.
Add "hello world" to main.rs

l```shell
trunk serve --open
```
The shell opens 127.0.0.1:8080 in the browser,
rendering a blank page.

**Expected behavior**
Rendered text saying "Hello, world!" at localhost:8080/

**Screenshots**
![leptos-trunk-serve](https://github.com/leptos-rs/leptos/assets/9915390/30d8b287-81f8-4d35-9172-2baa7d840a71)

![result](https://github.com/leptos-rs/leptos/assets/9915390/7feefb0e-6e3c-4454-9047-1aa6328cf497)

**Additional context**
rustup version 1.26.0
cargo version 1.72.0-nightly
trunk version 0.17.1
Browser: Microsoft Edge Version 114.0.1823.58 (Official build) (64-bit)

installed targets for active toolchain
--------------------------------------
wasm32-unknown-unknown
x86_64-unknown-linux-gnu

active toolchain
----------------
nightly-x86_64-unknown-linux-gnu (default)
rustc 1.72.0-nightly (330727467 2023-06-29)

**Comment by gbj** [comment]
Thanks. This was just fixed in the docs in #1256. Use `cargo add leptos --features=csr,nightly` (if you're using nightly Rust) or `cargo add leptos --features=csr` (if you're using stable). I changed the features and forgot to update that part of the book.

**Comment by jacksonbrim** [comment]
Awesome! Can't wait to check it out, now.
